pkg/st

```diff
@@ -14,25 +15,17 @@ libxext
 libxft
 
 [build]
-patch -p1 < "$K"/st-delete-key-works-always.patch
-patch -p1 < "$K"/st-no-bold-colors-20160727-308bfbf.patch
-patch -p1 < "$K"/st-scrollback-20160924-f739843.patch
-patch -p1 < "$K"/st-solarized-both-20160727-308bfbf.patch
-
-sed -i s/fdf6e3/ffeccf/g config.def.h #brwhite
-sed -i -e '1s/eee8d5/666666/;t' -e '1,/eee8d5/s//666666/' config.def.h #white in dark theme
-sed -i s/eee8d5/333333/ config.def.h #white in light theme
-sed -i s/859900/048103/g config.def.h #green
-sed -i s/657b83/000000/g config.def.h #bryellow
-sed -i s/2aa198/034b81/g config.def.h #cyan
-sed -i s/b58900/ff8b03/g config.def.h #yellow
-
+patch -p1 < "$K"/st-scrollback-0.8.patch
+patch -p1 < "$K"/st-scrollback-mouse-0.8.patch
+patch -p1 < "$K"/st-no-bold-colors-0.8.1.patch
+patch -p1 < "$K"/st-solarized-both-0.8.1.patch
+patch -p1 < "$K"/st-config.def.h-0.8.1.patch
 
 make \
  PREFIX="$butch_prefix" \
  CFLAGS="$optcflags -D_GNU_SOURCE -std=c99 -I$butch_root_dir$butch_prefix/include/freetype2 \
    -DVERSION=\\\"0.7.0\\\" -D_BSD_SOURCE -D_XOPEN_SOURCE=600" \
- LDFLAGS="$optldflags -lX11 -lXext -lXft -lfontconfig -lfreetype \
+ LDFLAGS="$optldflags -lm -lrt -lX11 -lutil -lXft -lfontconfig -lfreetype \
```

Comment:
how did you come up with those libs ? they're all inside musl's libc.so, so listing them here doesn't change anything

Reply:
I stole them from st's config.mk. you're right, they aren't necessary. How do you know what is included in libc by default? check `ldd` output for linking outside of libc.so?

Reply:
```
 $ la /opt/musl/lib/lib*
-rw-r--r--    1 root     root      10927022 Oct 13 15:26 libc.a
-rwxr-xr-x    1 root     root        583776 Oct 13 15:26 libc.so
-rw-r--r--    1 root     root             8 Oct 13 15:26 libcrypt.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 libdl.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 libm.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 libpthread.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 libresolv.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 librt.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 libutil.a
-rw-r--r--    1 root     root             8 Oct 13 15:26 libxnet.a
```
all files with size 8 are just empty dummy archives